set.mm

```diff
@@ -275107,6 +275229,741 @@ arbitrary neighborhoods (such as "locally compact", which is actually
       K $.
   $}
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+                  Refinements
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c Fne $.
+  $( Extend class definition to include the "finer than" relation. $)
+  cfne $a class Fne $.
+
+  $c Ref $.
+  $( Extend class definition to include the refinement relation. $)
+  cref $a class Ref $.
+
+  $c PtFin $.
+  $( Extend class definition to include the class of point-finite covers. $)
+  cptfin $a class PtFin $.
+
+  $c LocFin $.
+  $( Extend class definition to include the class of locally finite covers. $)
+  clocfin $a class LocFin $.
+
+  ${
+    $d x y z $.
+    $( Define the fineness relation for covers.  (Contributed by Jeff Hankins,
+       28-Sep-2009.) $)
+    df-fne $a |- Fne = { <. x , y >. | ( U. x = U. y /\ A. z e. x z C_ U. ( y
```

Comment:
Is this a standard notion ?

Reply:
Here I chose to move the whole section from Jeff Hankins' mathbox to the main part. You're right, I think while "refinement" is a standard notion, the "finer" relation for covers is not. What do others think? I could move this back into Jeff's mathbox.